python/sglang/srt/model_executor/model_runner.py

```diff
@@ -1432,6 +1433,12 @@ def _get_attention_backend_from_str(self, backend_str: str):
             )
 
             return CutlassMLABackend(self)
+        elif self.server_args.attention_backend == "trtllm_mla":
+            if not self.use_mla_backend:
+                raise ValueError("trtllm_mla backend can only be used with MLA models.")
```

Comment:
In the follow up FP8 PRs, let's add an SM100 check here so that the backend is not expected to work for <SM100